src/VisualStudio/IntegrationTest/TestUtilities/InProcess/ErrorList_InProc.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using EnvDTE80;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Common;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess
+{
+    internal class ErrorList_InProc : InProcComponent
+    {
+        public static ErrorList_InProc Create()
+            => new ErrorList_InProc();
+
+        public void ShowErrorList()
+            => ExecuteCommand("View.ErrorList");
+
+        public int ErrorListErrorCount
+            => GetErrorCount();
+
+        public void WaitForNoErrorsInErrorList()
+        {
+            while (GetErrorCount() != 0)
+            {
+                Thread.Yield();
+            }
+        }
+
+        public void NavigateToErrorListItem(int itemIndex)
+        {
+            var errorItems = GetErrorItems().AsEnumerable();
+            if (itemIndex > errorItems.Count())
+            {
+                throw new ArgumentException($"Cannot Navigate to Item '{itemIndex}', Total Items found '{errorItems.Count()}'.");
+            }
+            errorItems.ElementAt(itemIndex).Navigate();
+        }
+
+        public int GetErrorCount()
+        {
+            var errorItems = GetErrorItems();
+            try
+            {
+                return errorItems
+                    .AsEnumerable()
+                    .Count();
```

Comment:
Why not just use the [property](https://msdn.microsoft.com/en-us/library/envdte80.erroritems_properties.aspx)?